server/services/timelineService.ts

```diff
@@ -0,0 +1,113 @@
+import moment from 'moment'
+import type { Licence, CvlPrisoner } from '../@types/licenceApiClientTypes'
+import LicenceApiClient from '../data/licenceApiClient'
+import { convertToTitleCase } from '../utils/utils'
+import LicenceStatus from '../enumeration/licenceStatus'
+import LicenceType from '../enumeration/licenceType'
+import { User } from '../@types/CvlUserDetails'
+import LicenceEventType from '../enumeration/licenceEventType'
+import TimelineEvent from '../@types/TimelineEvent'
+import TimelineEventType from '../enumeration/TimelineEventType'
+import LicenceKind from '../enumeration/LicenceKind'
+
+export default class TimelineService {
+  constructor(private readonly licenceApiClient: LicenceApiClient) {}
+
+  private async getReviewEvents(licence: Licence, user: User): Promise<TimelineEvent[]> {
+    const conversationEventTypes = [LicenceEventType.HARD_STOP_REVIEWED_WITHOUT_VARIATION.valueOf()]
+    const events = await this.licenceApiClient.matchLicenceEvents(
+      `${licence.id}`,
+      conversationEventTypes,
+      'eventTime',
+      'DESC',
+      user
+    )
+    return events.map(
+      e =>
+        new TimelineEvent(
+          TimelineEventType.REVIEWED_WITHOUT_VARIATION,
+          'Licence reviewed without being varied',
+          licence.statusCode,
+          convertToTitleCase(`${e.forenames} ${e.surname}`),
+          licence.id,
+          e.eventTime
+        )
+    )
+  }
```

Comment:
new function to create timeline events from licence events (there will only ever be one per licence)

These licence events are then concat-ed to the standard time line events and the whole lot is sorted by time, most recent first

(the standard time line events are a bit wrong and misleading - haven't fixed those)